### ISSUE TITLE
Accessibility: Text editor formatting options issues

### ISSUE THREAD

**Issue by kevinkidwell** [cul-it/exhibits-library-cornell-edu#424]
**Description**
Text block creation seems to use non-compliant code when applying Bold and Italics.

This can be seen when you edit an exhibit page with a text box or create a new page with one. Highlighting text will display a tooltip with options to format text.

**Solution**
When the text box formatting options make text bold or italic, it should be using the following:
Bold: `<strong>`
Italic: `<em>`
